kilkari/src/main/java/org/motechproject/nms/kilkari/domain/InboxCallData.java

```diff
@@ -0,0 +1,119 @@
+package org.motechproject.nms.kilkari.domain;
+
+import org.joda.time.DateTime;
+import org.motechproject.mds.annotations.Entity;
+
+/**
+ * Data specific to the call contents (ie: what sound file was played) sent by the IVR system as part of
+ * {@link org.motechproject.nms.kilkari.domain.InboxCallDetails}
+ */
+@Entity
+public class InboxCallData {
```

Comment:
I thought it would work but it turns out Jackson (or someone else in the magic chain) can't handle inner classes.
So I had to put them back in separate files.
:sob: